packages/dhis2/src/meta/config.js

```diff
@@ -1,7 +1,7 @@
 // Config for dhis2 sandbox
 export default {
   configuration: {
-    hostUrl: 'https://play.im.dhis2.org/stable-2-42-1',
+    hostUrl: 'https://play.im.dhis2.org/dev-2-40',
```

Comment:
Shouldn't it just be `/dev`? The idea of this was that the URL would never have to change

See https://play.im.dhis2.org/dev/login/

Reply:
Fixed